client/src/plus/common/api.tsx

```diff
@@ -255,6 +256,18 @@ export function useFrequentlyViewedData(searchTerms: string) {
   return { data, setFrequentlyViewed };
 }
 
+export async function getStripePlans() {
+  let res;
+  //This comes from edge lambda so must be from live.
+  if (window.location.hostname.includes("localhost")) {
+    res = await fetch("https://developer.allizom.org/api/v1/stripe/plans");
+  } else {
+    res = await fetch(STRIPE_PLANS_PATH);
```

Comment:
We'll have to decide what to do about offline.

Reply:
let's just do what we do with the Watch and collections buttons and show a go online label or something similar.